### PR TITLE
[DEM-928] As an operator i want to measure according to Elzerman readout

### PR DESCRIPTION
**Pull request by qSaevar**
🌴

### CODE REVIEW

File: docs/notebooks/unfinished/example_measure_elzerman_readout.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+from qtt.measurements.acquisition import UHFLIScopeReader, UHFLIStimulus
```
Comment (peendebak) [inline docs/notebooks/unfinished/example_measure_elzerman_readout.py]: Rename the file to something else, the example is about triggered readout of the device, but not related to Elzerman.

File: docs/notebooks/unfinished/example_measure_elzerman_readout.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+# INITIALIZE THE SCOPE READER
+
+device_id = 'dev2338'
+time_period = 2
+trigger_enabled = True
+input_channel = 1
+input_channel_attribute = 'Demod 1 R'
+scope_reader = UHFLIScopeReader(address=device_id)
+sampling_rate = scope_reader.get_nearest_sample_rate(27e3)
```
Comment (peendebak) [inline docs/notebooks/unfinished/example_measure_elzerman_readout.py]: The `sampling_rate` is not used in the script. Should it be set on the `UHFLIScopeReader`?

File: docs/notebooks/unfinished/example_measure_elzerman_readout.py (hunk 1 above)
Comment (peendebak) [inline docs/notebooks/unfinished/example_measure_elzerman_readout.py]: This example script cannot be executed without the UHFLI hardware. If this is converted to a jupyter notebook, then the output that is generated by the script can also be included. E.g. the output of `print(records)`

Would it be an option to do that?

Comment (CumulonimbusCalvus) [reply docs/notebooks/unfinished/example_measure_elzerman_readout.py]: Maybe we can add the Elzerman related stuff next week and turn it into a full sample? We could then also convert the sample into a notebook.

**Pull request merged**